app/services/NotificationService.scala

```diff
@@ -52,8 +53,16 @@ class NotificationService @Inject()(val amlsNotificationConnector: AmlsNotificat
       case ContactType.ApplicationApproval => handleEndDateWithRefMessage(amlsRegNo, id, contactType, templateVersion, accountTypeId)
 
       case ContactType.RenewalApproval |
-           ContactType.AutoExpiryOfRegistration |
-           ContactType.RenewalReminder => handleEndDateMessage(amlsRegNo, id, contactType, templateVersion, accountTypeId)
+           ContactType.AutoExpiryOfRegistration => handleEndDateMessage(amlsRegNo, id, contactType, templateVersion, accountTypeId)
+
+      case ContactType.RenewalReminder |
+           ContactType.NewRenewalReminder => {
+        templateVersion match {
+          case "v5m0"  => handleEndDateMessage(amlsRegNo, id, contactType, templateVersion, accountTypeId)
+          case "v1m0" | "v2m0" | "v3m0" | "v4m0" | "v5m0 " => handleEndDateMessage(amlsRegNo, id, contactType, templateVersion, accountTypeId)
```

Comment:
This can't be right, you have included "v5m0" in this case too (I know it want ever match there, because it is lower, but anyway ...)

More importantly both matches return the same value:

handleEndDateMessage(amlsRegNo, id, contactType, templateVersion, accountTypeId)

What do you mean to do?

Reply:
changes reverted here -> https://github.com/hmrc/amls-frontend/pull/1516/commits/115f2a10c48572c6bbbf6ac6ceb7070c7b5d509b